Module 1/DS/HashMap.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Set;
+import java.util.HashSet;
+
+class HashMap {
```

Comment:
In HashMap as well, we intend to have a menu-driven program for add, delete, update, sort and search.

Reply:
Yes ma'am I will change it.